crates/cairo-coverage/tests/data/readme_example/Scarb.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "readme_example"
+version = "0.1.0"
+edition = "2024_07"
+
+# See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
+[dependencies]
+starknet = ">=2.8.0"
+
+[dev-dependencies]
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry", tag = "v0.29.0" }
+
+[[target.starknet-contract]]
+sierra = true
+
+[scripts]
+test = "snforge test"
+
+[cairo]
```

nit: To align with the recommended configuration
```suggestion
[profile.dev.cairo]
```